test/fs/test_mount.c

```diff
@@ -51,23 +74,44 @@ int main() {
     } catch (e) {
       ex = e;
     }
-    assert(ex instanceof FS.ErrnoError && ex.errno === 28); // EINVAL
+    assert(ex.name === 'ErrnoError' && ex.errno === 28); // EINVAL
 
     // mount and unmount again
     FS.mount(MEMFS, {}, '/working');
     FS.unmount('/working');
 
+    ex = null;
     // try to read the file from the old mount
     try {
       FS.readFile('/working/waka.txt', { encoding: 'utf8' });
     } catch (e) {
       ex = e;
     }
-    assert(ex instanceof FS.ErrnoError && ex.errno === 44); // ENOENT
+#if !defined(WASMFS)
+      // WasmFS readFile aborts on failure, instead throwing an ErrnoError.
```

Comment:
Is this something we should fix in a follow-up PR? If so, it might be good to have a TODO here and maybe even file an issue about it if it's not going to be fixed soon.